### PR TITLE
added imlib checks

### PR DESCRIPTION
**Pull request by dagelf**
Prevent imlib logging a lot of errors to the console for icons that don't work...

### CODE REVIEW

File: xlunch.c [hunk 1]
```diff
@@ -1147,8 +1147,8 @@ int main(int argc, char **argv)
                       int d;
                       if (current->clicked) d=2; else d=0;
 
-                      imlib_blend_image_onto_image(image, 0, 0, 0, w, h,
-                                                  current->x - up_x + cell_width/2-icon_size/2+d, current->y - up_y +padding+margin+d, icon_size-d*2, icon_size-d*2);
+                      if (image) { imlib_blend_image_onto_image(image, 0, 0, 0, w, h,
```
Comment (Tomas-M) [inline xlunch.c:1150]: Hm, this is already in if (image) block, why would this help?

**Review by Tomas-M — CHANGES_REQUESTED** [review]
I am not sure if all the modifications are really necessary.

**Comment by dagelf** [comment]
That one's unnecessary, the others not as they prevent asking imlib to free
nulls, littering the error log with hundreds of messages. A better fix
would be to break earlier if null images are returned.

Op 15 Jul 2017 22:11, het "Tomas-M" <notifications@github.com> geskryf:

*@Tomas-M* requested changes on this pull request.

I am not sure if all the modifications are really necessary.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/Tomas-M/xlunch/pull/27#pullrequestreview-50196754>, or mute
the thread

**Comment by Tomas-M** [comment]
I'm lazy to write it better so I just merged your patch. Thank you;

**Pull request merged**